pkg/queue/breaker_test.go

```diff
@@ -187,6 +187,30 @@ func TestBreaker_UpdateConcurrency(t *testing.T) {
 	}
 }
 
+func TestBreaker_Timeout(t *testing.T) {
```

Comment:
```suggestion
func TestBreakerTimeout(t *testing.T) {
```

Reply:
Though other tests follow this strange pattern. May be I'll just rename them in bulk.

Reply:
+1 lets do them all at once